src/app/recent/[...regionCodeSlug]/page.tsx

```diff
@@ -52,11 +52,9 @@ const getRecentSightings = async (
   });
 
   return response.json();
-};
+}
 
-const getBirdImage = async (
-  speciesName: string,
-): Promise<{
+async function getBirdImage(speciesName: string): Promise<{
```

Comment:
turned these into functions. I personally prefer to use the actual function declaration wherever possible, it tends to result in nicer stack trace and generally easier to scan code when all functions have the word `function` on them.

Reply:
Nice! I'm currently very arrow function pilled from how I've learned React so I'll keep that in mind.